### ISSUE TITLE
Acute and Omlut diacritics do not print properly

### ISSUE THREAD

**Issue by rbann11** [groverburger/sapling#2]
When attempting to print  accented characters with acutes or Omluts the word Dead then the character is printed to the screen.

Example: é is printed DEADe.

On US international keyboard the single and doubles quotes are used as dead keys to create these characters.

I hope is helps,
   Roger

**Comment by groverburger** [comment]
Interesting, thanks for pointing this out.

I wonder if this is a problem with the font... I'll look into it.

**Comment by amunn** [comment]
I don't think this is font related because if you hard code an é into the character parsing code it displays correctly.  So I think it's simply ignoring the dead keys altogether. On a Mac keyboard (I use the Canadian English layout), I don't get `DEADe`, I just get `e`.  The Mac uses Option-e e for e.g. é with an accute accent, Option-u o for e.g. ö and none of these display.
